Sources/Testing/Expectations/Expectation.swift

```diff
@@ -113,7 +113,7 @@ extension Expectation {
 
     /// Creates a snapshot expectation from a real ``Expectation``.
     /// - Parameter expectation: The real expectation.
-    init(expectation: Expectation) {
+    init(snapshotting expectation: Expectation) {
```

Comment:
Okay, if we _are_ using `snapshotting`, we should update `Expectation.snapshot.init(expectation:)` to match.

Reply:
That's what that is? I think?

Reply:
I must have drank decaf this morning… sorry!

Reply:
No problem at all, I was just a bit confused :)